x-pack/plugins/alerting/server/rules_client/tests/disable.test.ts

```diff
@@ -325,6 +329,13 @@ describe('disable()', () => {
         kind: 'alert',
       },
       kibana: {
+        alert: {
+          rule: {
+            execution: {
+              uuid: 'rules-client', // TODO: See TODO in `rules_client.ts`
```

Comment:
This'll need updated based on the feedback from: https://github.com/elastic/kibana/pull/113058/files#r771762662